demo-app/src/main/java/com/jfcbuilder/demo/JFreeChartBuilderDemo.java

```diff
@@ -218,6 +251,8 @@ private static JFreeChart multiPlotMinuteTimeSeries() {
       .timeData(timeArray)
       .indexRange(startIndex, endIndex)
 
+      // Deprecation note: preserve a couple of calls to gridLines() and noGridLines() for testing until they're removed.
```

Comment:
In hindsight, keeping `noGridLines()` is a handy semantic facility to turn off all grid lines regardless of underlying grid customization logic. Revert the deprecation of `noGridLines()`!